modules/pagerduty-team/inputs.tf

```diff
@@ -0,0 +1,9 @@
+variable "description" {
+  description = "A description of the team. If not set, a placeholder of `Managed by Terraform` will be set."
+  type        = string
+}
+
+variable "name" {
+  description = "The name of the team."
+  type        = string
+}
```

Comment:
do you want to create subteam? https://registry.terraform.io/providers/PagerDuty/pagerduty/latest/docs/resources/team#parent

Reply:
and I think we can add membership modules (3 modules) to this module too

memberships_observer = [user_id1,user_id1]
memberships_responder = [user_id1,user_id1]
memberships_manager = [user_id1,user_id1]

Reply:
I added now input for members. I think no need to have three separate inputs. Just one `members` is enough.